### DISCUSSION TITLE
ionizable chemical groups

### DISCUSSION THREAD

**Discussion by adrian-jasinski** [rdkit/rdkit#6173] [Q&A] [answered]
How can I find ionizable chemical groups (acid or base) using RDKit?

How can I mark them on molecule graphics/image?

**Comment by greglandrum** [comment, marked as answer]
Hi @admed. In order to do this you would need to have either a pKa calculator, a model of some kind that can predict likely ionizable sites, or a set of SMARTS patterns which flag groups that are likely to be ionizable. There are papers out there describing all of the above, but the RDKit doesn't have anything like this built in that I would recommend using.
